utils/constants.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
     MaxParallelLimit: 10,
+    frequency: 30,
+    threshold: 5,
```

Comment:
* these parameters are unintellegible → make them explicit (`prometheusPollingFrequency`...), or just make the computation once and have just a "final scrapping timeout" variable
* the values should not be hard-coded here, but set by ZKOP: i.e. passed via env variables (though we may have defaults obviously)

→ should be top-level const in `CountItems/masterProcess.js`

Reply:
done here : https://github.com/scality/s3utils/pull/312/commits/a50c483216bfb00c5fa7bc9b25c02e1cbb673bbb#diff-46cbf734fd7b9022c7bbf597a8a5deb96829b581ca817a861bbc3bf1e86274a9R25-R32